eks-distro-base/scripts/eks-d-common

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/env bash
```

Comment:
I have set up this common script for use by my next PR which will use this same ldd approach for installing packaging.